.circleci/config.yml

```diff
@@ -325,38 +325,32 @@ jobs:
             fi
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
-          name: 'Generate Version Number'
-          command: echo "export VERSION=$(npm run --silent get-next-version | xargs)" >> $BASH_ENV
-      - run: echo ${VERSION}
+          name: 'Store current version number'
+          command: echo "export PREVIOUS_VERSION=$(npm run --silent tooling -- version current | xargs)" >> $BASH_ENV
       - run:
-          name: 'Build Packages'
-          command: npm run build
-      - run:
-          name: 'Build UMD script'
-          command: npm run build:script
-      - run:
-          name: 'Bump all package version'
-          command: npm run tooling -- version set ${VERSION} --all
-      - run:
-          name: 'Commit bumped version'
-          command: |
-            git add packages/node_modules/*/package.json packages/node_modules/@ciscospark/*/README.md packages/node_modules/@webex/*/package.json packages/node_modules/webex/umd/*.js
-            git commit -m "chore(release): v${VERSION} [skip ci]"
+          name: 'Run standard-version'
+          command: npm run release
       - run:
           name: 'Push to upstream/master'
           command: git push upstream HEAD:master
       - run:
-          name: 'Tag for release'
-          command: git tag "v${VERSION}"
-      - run:
-          name: 'Push tagged release to upstream/master'
-          command: git push upstream v${VERSION}:v${VERSION}
-      - run:
-          name: 'Generate Package Dependencies'
-          command: npm run deps:generate
+          name: 'Push tag to upstream/master'
+          command: git push --tags upstream
```

Comment:
I think this should happen after we publish to NPM. If for whatever reason NPM publishing fails, we would already have pushed the new release in Git, and I think there's more clean up to do then, or may run into situations where we have to make exceptions in the whole flow because nothing but publishing has to happen again.

This is to say that I think is safer/easier to push the new version changes to Git only after publishing is successful. That's how we [do it in the components](https://github.com/webex/components/blob/master/release.config.js) too.

Reply:
I can do that and it makes sense
Most of these tools suggest you push w/ tags first then publish

Reply:
I'm taking it from the suggested flow [from semantic-release](https://semantic-release.gitbook.io/semantic-release/usage/plugins#plugins-declaration-and-execution-order), but I think it makes sense for all the other reasons I mentioned too 😁

Reply:
Aight i googled some more, im seeing more examples that use npm publish -> git push
So i'll switch to that